pkg/maps/ctmap/ctmap.go

```diff
@@ -264,6 +268,46 @@ func (m *Map) DumpEntries() (string, error) {
 	return DoDumpEntries(m)
 }
 
+// DumpAll batch dumps the Map m and returns the count of the entries.
+func (m *Map) DumpAll() (int, error) {
+	// TODO: Preallocating like this can put quite a bit of pressure on Go GC
+	// presumably, could we do something about that?
+	mx := m.MaxEntries()
+	vout := make([]CtEntry, mx)
```

Comment:
@ti-mo I recall you were looking into batch lookups for CT map. Do you have any thoughts about the above TODO?

Reply:
Good call out! I've implemented something similar to `DumpReliablyWithCallback` and wrote a unit test to confirm it.